h/accounts/schemas.py

```diff
@@ -63,7 +63,9 @@ def email_node(**kwargs):
         validator=colander.All(
             validators.Length(max=EMAIL_MAX_LENGTH), validators.Email(), unique_email
         ),
-        widget=deform.widget.TextInputWidget(template="emailinput"),
+        widget=deform.widget.TextInputWidget(
+            template="emailinput", autocomplete="username"
```

Comment:
I think this could be "email" instead of "username", and above, I think we could add autocomplete="username" for `unique_username`